cpu/em_smtfolding.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+#
+# Copyright: 2018 IBM
+# Author: Shriya Kulkarni <shriyak@linux.vnet.ibm.com>
+import os
+from avocado import Test
+from avocado import main
+from avocado.utils import archive
+from avocado.utils import process
+from avocado.utils import build
+from avocado.utils import process, cpu
+from avocado.utils.software_manager import SoftwareManager
+
+
+class smt_folding(Test):
+    """
+    Throughput test for SMT folding in presence of swizzle.
+    """
+    def setUp(self):
+        '''
+        1.Build ebizzy
+        '''
+        sm = SoftwareManager()
```

Comment:
can you please have variable name at least three char

Reply:
Sure, I will make those change.